engine/function/src/templates/Sort.ftl

```diff
@@ -202,6 +318,63 @@ public class Sort {
         return vs;
     }
 
+    /**
+     * Returns the indices of values sorted from smallest to largest.
+     *
+     * @param values values.
+     * @return sorted indices.
+     */
+    public static int[] sortIndex(final ${pt.vector} values) {
```

Comment:
I'd prefer calling this `rank()`, since it's shorter and it's the name both pandas and kx use